agent/engine/docker_container_engine.go

```diff
@@ -452,11 +455,11 @@ func (dg *dockerGoClient) stopContainer(ctx context.Context, dockerId string) Do
 		return DockerContainerMetadata{Error: CannotGetDockerClientError{version: dg.version, err: err}}
 	}
 
-	err = client.StopContainer(dockerId, dockerStopTimeoutSeconds)
+	err = client.StopContainer(dockerId, uint(dg.config.DockerStopTimeoutSeconds))
```

Comment:
There is an error here.
If `dg.config.DockerStopTimeoutSeconds` is a `time.Duration` object,
it should be used here as `uint(dg.config.DockerStopTimeoutSeconds/time.Second)`, otherwise the function will get # of microseconds instead.